fastlane/Fastfile

```diff
@@ -199,28 +199,46 @@ platform :ios do
     # Ask for new version number
     new_version_number = UI.input("New version number: ")
 
+    changelog_path = edit_changelog
+
+    create_new_release_branch(version: new_version_number)
+    replace_version_number(version: new_version_number)
+    commit_updated_files_and_push(version: new_version_number)
+
+    changelog = File.read(changelog_path)
+    create_pull_request(
+      title: "Release/#{new_version_number}",
+      base: "main",
+      body: changelog
+    )
+  end
+
+  private_lane :edit_changelog do |options|
     # Open CHANGELOG.latest.md in editor
     changelog_filename = "CHANGELOG.latest.md"
     changelog_path = File.absolute_path("../#{changelog_filename}")
     editor = ENV['FASTLANE_EDITOR'] || ENV['EDITOR'] || 'vim'
 
+    content_before_opening_editor = File.read(changelog_path)
+
     UI.message("Will use '#{editor}'... Override by setting FASTLANE_EDITOR or EDITOR environment variable")
     if UI.confirm("Open #{changelog_filename} in '#{editor}'?")
-      system(editor, File.absolute_path(changelog_path).shellescape)
+      system(editor, changelog_path.shellescape)
     else
       UI.user_error!("Cancelled")
     end
 
-    create_new_release_branch(version: new_version_number)
-    replace_version_number(version: new_version_number)
-    commit_updated_files_and_push(version: new_version_number)
+    # Some people may user visual editors and `system` will continue right away
+    # This will compare the content before and afer attempting to open and
+    # and will open a blocking prompt for the visual editor changes to be saved
+    content_after_opening_editor = File.read(changelog_path)
+    if content_before_opening_editor == content_after_opening_editor
+      unless UI.confirm("You may have opened the changelog in a visual editor. Enter 'y' when changes are saved or 'n' to cancel")
```

Comment:
`.` at end of sentence

Reply:
None of the others have `.` at the end 🤔  Still want this?

Reply:
We probably should try to use sentence capitalization. At least for consistency

Reply:
Should I add/fix the punctuation everywhere then? 😊

Reply:
I'd be down for that, but if you don't have the time, definitely just do your additions 😄 We can update the rest later

Reply:
I'll wait! I'm unsure how I feel about it because its _fastlane_ style guidelines to not have punctuation at the end of `UI` thingies 🤷‍♂️

Reply:
OOooo yeah, ok, that's weird. Let's just do the comments, then.